docs/detections/api/rules/rules-api-bulk-actions.asciidoc

```diff
@@ -265,6 +276,14 @@ to apply.
 Yes, if action is `edit`.
 |==============================================
 
+[[bulk-actions-rules-api-dry-run]]
+[discrete]
+==== Dry run mode
+Enable dry run mode before you bulk update or delete rules to verify that the selected rules can be modified _before_ you permanently change them. When dry mode is enabled, the bulk action is temporarily applied to selected rules. These updates are not written to {es}.
```

Comment:
@banderror @vitaliidm should we tweak the first sentence a bit more? Looking at this again, I'm not sure that it fully encompasses the main use cases that this feature addresses. Here's my understanding:
- The dry run allows users do a test run of the bulk action they select (enable, disable, delete, etc.) to help them verify that the rules they specified can actually be updated. 
- Users can complete a pelimiary and temporary bulk rule update with this feature. This would give them an extra buffer layer, or another layer of checks, before they permanently alter their rules.

Reply:
If the above is correct, here's another revision that includes both. Let me know what you think:

```suggestion
Enable dry run mode to allow test runs when applying bulk actions to rules. Dry runs can help you verify that the rules you specified can actually be updated. They're also useful for testing bulk rule updates before they're permanently applied. When dry mode is enabled, bulk actions are temporarily applied to specified rules to show you a realistic set of results. Rule updates are not written to {es}.
```

Reply:
@nastasha-solomon I don't think that strictly speaking this is a correct statement:

> When dry mode is enabled, bulk actions are temporarily applied to specified rules to show you a realistic set of results.

In the dry run mode, we don't return any results in the response body. We only return errors if there are any. So it's correct to say that this mode "can help you verify that the rules you specified can actually be updated". But it won't help the user to preview what would be the result (the updated, created, or deleted rules) of an action. So, you can think about this mode as a validation function.

Reply:
@banderror ah, thanks for clarifying that. Would it be correct to say that dry run mode identifies rules that will not accept bulk actions?

Reply:
@nastasha-solomon yep, that would be correct!